Code/GraphMol/resMolSupplierTest.cpp

```diff
@@ -251,6 +250,7 @@ void testZwitterion() {
   delete resMolSuppl;
   resMolSuppl = new ResonanceMolSupplier(
       (ROMol &)*mol, ResonanceMolSupplier::ALLOW_CHARGE_SEPARATION);
+  std::cerr << "resMolSuppl->length() " << resMolSuppl->length() << std::endl;
```

Comment:
probably don't need this anymore

Reply:
Yes, that's clearly a leftover I will get rid of.